src/controller/clients/shape-search-client.js

```diff
@@ -52,11 +52,13 @@ class ShapeSearchClient extends BaseClient {
     for (let unit in units) {
       // console.log("Unit: ", units[unit])
       for (let i in mObj) { mObj[i] = ""  }
-      let mObjPush = JSON.parse(JSON.stringify(mObj))
-      mObjPush.location = units[unit][0].location
-      mObjPush.condo = true
-      mObjPush.pwd_parcel_id = units[unit][0].pwd_parcel_id
-      data.rows.push(mObjPush)
+      let mObjPush = JSON.parse(JSON.stringify(mObj));
+      mObjPush.owner_1 = "Condominium (" + units[unit].length + " Units)";
```

Comment:
Why are we putting `Condomium NNN Units` as the text here? Also, why is the same string being created in so many different places?

Reply:
I'm adding this text into the owner field since it was previously blank. I wanted to both fill the empty fields, and find a place to permanently display this information in the table.